app/Controllers/feedController.php

```diff
@@ -136,7 +149,7 @@ public function addAction() {
 				// User want to create a new category, new_category parameter
 				// must exist
 				$new_cat = Minz_Request::param('new_category');
-				$new_cat_name = isset($new_cat['name']) ? $new_cat['name'] : '';
+				$new_cat_name = isset($new_cat['name']) && !empty(trim($new_cat['name'])) ? trim($new_cat['name']) : '';
```

Comment:
`empty(function())` is only accepted since PHP 5.5 http://php.net/manual/function.empty.php

I think the following should be sufficient:

```php
$new_cat_name = isset($new_cat['name']) ? trim($new_cat['name']) : '';
```